docs/website/content/docs/installation.md

```diff
@@ -10,7 +10,7 @@ toc: true
 
 Sysl is a CLI (Command Line Interface) that excecutes with the `sysl` command.
 
-    go get -v github.com/anz-bank/sysl/cmd/sysl
+    GO111MODULE=on go get github.com/anz-bank/sysl/cmd@v0.4.0
```

Comment:
should be `.../sysl/cmd/sysl@v0.4.0`. i.e. needs more sysl

Reply:
Done 🤦‍♀